project/Dependencies.scala

```diff
@@ -141,6 +141,8 @@ object Dependencies {
     workbenchGoogle2Test,
     workbenchOpenTelemetry,
     workbenchOpenTelemetryTest,
+    helmScalaSdk,
```

Comment:
Should we remove these from `rootDependencies` since they're now in `coreDependencies`?

Reply:
`rootDependencies` correspond to `http`?

Reply:
yea, it's carry over name when leo was previously non multi project...feel free to rename it to HTTP...you can probably remove https://github.com/DataBiosphere/leonardo/blob/develop/build.sbt#L8 as well